packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -340,6 +348,49 @@ export default class VList {
         );
     }
 
+    /**
+     * Change list style of the given range of this list.
+     * If some of the items are not real list item yet, this will make them to be list item with given style
+     * @param start Start position to operate from
+     * @param end End position to operate to
+     * @param targetStyle Target list style
+     */
+    setListStyleType(
+        start: NodePosition,
+        end: NodePosition,
+        targetStyle:
+            | NumberingListType
+            | BulletListType
+            | CompatibleBulletListType
+            | CompatibleNumberingListType
+    ) {
+        this.findListItems(start, end, item => {
+            item.getNode().title = targetStyle.toString();
```

Comment:
Title prop adds a tooltip to the List Item, I wonder if we can find a way to instead of using this property to every List Item we can just apply use the metadata api and just apply it to the rootList.
That way we only add a single property to the List instead of adding properties to all the items.